src/bioregistry/data/curated_mappings.sssom.tsv

```diff
@@ -99,3 +99,103 @@ bioregistry:uniprot.ptm	Not	skos:exactMatch	prefixcommons:ptmswitchboard	orcid:0
 bioregistry:vso	Not	skos:exactMatch	lov:vso	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:wikipathways	Not	skos:exactMatch	aberowl:wikipathways	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
 bioregistry:wikipathways	Not	skos:exactMatch	bioportal:wikipathways	orcid:0000-0003-4423-4370	semapv:ManualMappingCuration	
+bioregistry:uniprot	Not	skos:exactMatch	aberowl:UP	orcid:0009-0009-5240-7463	semapv:ManualMappingCuration	
```

Comment:
As I noted in #1457, I think `semapv:MappingReview` is probably more appropriate for these curations

Reply:
There also seems to be a column count issue, and the file has to be linted